.stylelintrc.yml

```diff
@@ -0,0 +1,84 @@
+# We are using stylelint-config-standard for all of the base rules designed more broadly
+# for CSS. The stylelint-scss plugin provides us with more SCSS specific rules. Those
+# rules are prefixed by `scss/`
+extends: stylelint-config-standard
+plugins:
+   - stylelint-scss
+rules:
+   # Turn off the following rule since our ltr/rtl approach generally violates it.
+   no-descending-specificity: null
+   # Turn off config-standard rule in favor of the scss/at-rule-no-unknown rule
+   at-rule-no-unknown: null
+   scss/at-rule-no-unknown: true
+   # General rules
+   indentation: 3
+   linebreaks: unix
+   no-eol-whitespace: true
+   no-missing-end-of-source-newline: true
+   no-empty-first-line: true
+   max-line-length:
+      - 140
+      - ignorePattern:
+         - /https?://[0-9,a-z]*.*/
+   max-empty-lines: 1
+   string-quotes: single
+   string-no-newline: true
+   color-hex-case: lower
+   color-hex-length: long
+   color-no-invalid-hex: true
+   no-invalid-position-at-import-rule: null
```

Comment:
This is added from our existing config. Was there a known need or just something noticed that would be helpful?

Reply:
See note here: https://github.com/silvermine/standardization/pull/23#issuecomment-839125472

Reply:
🤦 lol thanks